content/docs/core-functions/moderation.md

```diff
@@ -3,27 +3,47 @@ title: Moderation
 
 ## On solutions to Power mods
 Solutions to ensure fair moderation often center the voices of the individuals in each community and give them the authority to govern themselves. We want this community to be able to govern itself at multiple levels. However self-governance is difficult. 
 
 ### Elections
-Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
+Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature an election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
 
 ### Sortition
-Sortition may offset some of these problems but also has its pros and cons. Perhaps most importantly, we don’t want to burden anyone who’s not interested in moderating with that responsibility. We think people would be well served to examine the social groups which exist in their lives which aren’t governed via direct democracy and to spend a bit of time considering what model is most appropriate in each sphere of their life. 
+Sortition (appointment by lot) may offset some of these problems but also has its pros and cons. Perhaps most importantly, we don’t want to burden anyone who’s not interested in moderating with that responsibility. We think people would be well served to examine the social groups which exist in their lives which aren’t governed via direct democracy and to spend a bit of time considering what model is most appropriate in each sphere of their life.
 
 ### Legitimacy
 We wouldn't want to receive medical care from an individual which was elected by the vote of non-medical professionals. The same would be true of legal advice. We would like our moderators to be educated and skilled in moderating, and both elections and sortition don’t always center these values. Ultimately, we haven’t decided on a sustainable long-term solution to moderation, and have been choosing active members in communities which seem to embody or align with our ethos to elevate to a moderating position.
+
+## Moderation at Beehaw
```

Comment:
Will ask if we wanna include this in docs

Reply:
Let me know if you want me to strip it back out.